client/src/app/tabs/panel/tabs/LintingTab.js

```diff
@@ -110,16 +110,34 @@ function getName(issue) {
   return id.toLowerCase();
 }
 
-function sortIssues(issues) {
-  return issues.sort((a, b) => {
-    a = getName(a),
-    b = getName(b);
-
-    if (a === b) {
-      return 0;
-    } else if (a < b) {
+/**
+ * Sort reports by:
+ *
+ * 1. category
+ * 2. name or ID
+ *
+ * @param {Object[]} reports
+ *
+ * @returns {Object[]}
+ */
+function sortReports(reports) {
+  return reports.sort((a, b) => {
```

Comment:
Note that `sort` mutates the array, so we end up with changing the order of items in the reports prop.

Reply:
Good catch. Changed it.